.buildkite/pull-requests.json

```
@@ -10,5 +10,17 @@
         "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
         "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
         "fail_on_not_mergeable": true
+      },
+      {
+        "enabled": true,
+        "pipeline_slug": "fleet-server-package-mbp",
+        "allow_org_users": true,
+        "allowed_repo_permissions": ["admin", "write"],
+        "set_commit_status": false,
+        "build_on_commit": false,
+        "build_on_comment": true,
+        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/package$",
+        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/package$",

```

I would change the comment regex to avoid triggering this pipeline with the same words as the other pipeline. Let's make the comment specific for this pipeline.

One option: 

```suggestion
        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:package)\\W+(?:this|it))|^/package$",
        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:package)\\W+(?:this|it))|^/package$",

```